cadquery/occ_impl/exporters/__init__.py

```diff
@@ -37,6 +37,19 @@ class ExportTypes:
 ]
 
 
+class STEPExportMode:
```

Comment:
I'd propose to use literals and move this to the exporters/assy

Reply:
@adam-urbanczyk I'm not sure if you're thinking mypy literals or string literals. Is the latest commit what you were thinking? I'm not sure how to set the literal type except in the docstring either because kwargs is being used.

Reply:
I guess both, e.g. https://github.com/CadQuery/cadquery/blob/6c8030d26a355b42362b64d502ad06b791e6bb30/cadquery/occ_impl/exporters/__init__.py#L43

Reply:
> Is this resolved? Maybe I misunderstood. I think the request is to remove the class `ExportModes'.

I'm not sure. I tried to match how it was done elsewhere in the code, but may have missed the mark.

Reply:
Yes, this is resolved